lib/redactor.js

```diff
@@ -8,7 +8,8 @@ function redactor ({secret, serialize, wcLen}, state) {
   /* eslint-disable-next-line */
   const redact = Function('o', `
     if (typeof o !== 'object' || o == null) {
-      throw Error('fast-redact: primitives cannot be redacted')
+      if (this.strict) throw Error('fast-redact: primitives cannot be redacted')
```

Comment:
you can compose a different function based on `strict`, there is no need to pass this down to the generated function.

Reply:
ah makes sense